desktop/cmp/treemap/SplitTreeMapModel.js

```diff
@@ -0,0 +1,107 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2019 Extremely Heavy Industries Inc.
+ */
+import {HoistModel, managed} from '@xh/hoist/core';
+import {bindable, computed} from '@xh/hoist/mobx';
+import {throwIf} from '@xh/hoist/utils/js';
+import {isNil, isFunction, sumBy} from 'lodash';
+
+import {TreeMapModel} from './TreeMapModel';
+
+/**
+ * Todo
+ */
+@HoistModel
+export class SplitTreeMapModel {
+
+    //------------------------
+    // Immutable public properties
+    //------------------------
+    /** @member {GridModel} */
+    gridModel;
+    /** @member {function} */
+    filter;
```

Comment:
if we are calling the sections 'posModel' and 'negModel', this implies a metric with a sign that will be different on both sides.    Seems like we either stick with that, and specify a 'sectionMetric' here OR we do something more generic like a "regionFilter" accompanied by a "primaryRegionModel' and a 'secondaryRegionModel'.